keras_nlp/models/distil_bert/distil_bert_tokenizer.py

```diff
@@ -112,3 +114,11 @@ def set_vocabulary(self, vocabulary):
     @classproperty
     def presets(cls):
         return copy.deepcopy(backbone_presets)
+
+    def get_config(self):
+        config = super().get_config()
+        # In the constructor, we pass the list of special tokens to the
```

Comment:
Maybe we can shorten these comments a bit?
`del config["special_tokens"]  # Not configurable; set in __init__.`